doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Samsung-Artik-5/index.md

```diff
@@ -12,13 +12,21 @@ sort_idx: 50
 
 The guide explains how to cross-compile Kaa applications for Samsung Artik 5 based on the Kaa C++ endpoint SDK.
 
-# Configuring Artik 5 board
+## Configuring Artik 5 board
 
 If this is the first time you use the Artik board, you have to start with configuring your board.
 For this purpose, refer to [the official Artik getting started guide](https://developer.artik.io/documentation/getting-started-beta/).
 After finishing with the guide, you will be able to access your board through both a serial terminal and an ssh client.
 
-# Installing prerequisites
+## Installing prerequisites
+
+**All steps described in this guide were tested on:**
+
+ - **Host OS:** Ubuntu 14.04 LTS Desktop 64-bit.
+ - **Device:** [Samsung Artik 5](https://www.artik.io/modules/overview/artik-5/)
+ - **Target OS:** Fedora
```

Comment:
It would be nice to specify version here. If you don't know, it is fine. We can add this information after you test it. 
I am sure that you have tested this cross-compilation guide, however, I doubt that you have run the demo on Artik 5 board.